scripts/includes/common.sh

```diff
@@ -193,7 +201,7 @@ verify_st2() {
   st2 --version
   st2 -h
 
-  st2 auth $USERNAME -p $PASSWORD
+  st2 login -p $PASSWORD -w $USERNAME
```

Comment:
Yeah, we need to confirm login stuff works correctly (e.g. if you run it as sudo as a random user or if you run it directly as root).

Login caches token locally and it writes it for user who executed the script so we need to make sure it works as expected.